tools/oav-traffic-converter/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Microsoft Corporation
+
```

Comment:
Do we need an independent license?

Reply:
🤷‍♂️ 

I used `npm init` for the initial stage of this, and it came with it. I'm fine with deleting it, or I can mark it in the source file itself. 

Given this was already taken care of, I just accepted this.